python/pyspark/util.py

```diff
@@ -61,6 +62,26 @@ def _get_argspec(f):
     return argspec
 
 
+def majorMinorVersion(version):
```

Comment:
Since this affects Spark core, it might be nice to put this in a separate PR.  Also, shall we make this match the Scala API?

Reply:
> Also, shall we make this match the Scala API?
Do you mean to throw exception when unable to parse Spark version?